lti_consumer/tests/unit/test_lti.py

```diff
@@ -259,7 +262,8 @@ def test_verify_result_headers_verify_content_type_true(self, mock_log):
 
         with self.assertRaises(LtiError):
             self.lti_consumer.verify_result_headers(request)
-            self.assertTrue(mock_log.called)
+
+        assert mock_log.error.called
```

Comment:
Why is `mock_log.error` always called now?

Reply:
Great question @ormsbee. The coverage reports the original `self.assertTrue(mock_log.called)` was identified as a dead statement i.e. never reached. 

That's because `self.lti_consumer.verify_result_headers(request)` throws a an `LtiError` exception, so the next statement is never reached.

So I moved it outside the `with` statement. The other options was to delete the `mock_log` assertion.

Anyway tests are supposed to be stupid (in a good way) without branching, so subsequent statements should always be unconditional and unbranched since all of them must pass.